dts/bindings/ethernet/nxp,mcux-ethernet.yaml

```diff
@@ -0,0 +1,32 @@
+---
+title: NXP MCUXpresso Ethernet
+id: nxp,mcux-ethernet
+version: 0.1
+
+description: >
+    This binding gives a base representation of the NXP MCUXpresso Ethernet
+inherits:
+  !include ethernet.yaml
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "nxp,mcux-ethernet"
```

Comment:
This is describing the hardware, whereas mcux is a software concept. Change the constraint and filename to "nxp,kinetis-ethernet"